package.json

```diff
@@ -127,6 +127,7 @@
     "@types/lodash.clonedeep": "^4.5.4",
     "@types/react-grid-layout": "^0.16.7",
     "@types/recompose": "^0.30.5",
+    "@use-it/interval": "^0.1.3",
```

Comment:
I like to document in the PR description when I introduce a dependency, what it is about, it is well maintained...
It's always easier to introduce a new dependency than to remove one, so I am a little picky about that.

Otherwise this package seems well done :)

Reply:
good idea; added a section to PR description

Reply:
I like this. Because I questioned this too, looked it up and thought it seemed a good idea. But providing this in a PR would be a major ++